config.nix

```diff
@@ -15,7 +15,7 @@ rec {
 
   # true = use a github repository as the holonix base (recommended)
   # false = use a local copy of holonix (useful for debugging)
-  use-github = true;
+  use-github = false;
```

Comment:
Are you sure you want to be changing this flag?  Otherwise looks good.

Reply:
@zippy nope, this was only temporary while i was cross testing things